packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -1,11 +1,24 @@
 import logging
+from subprocess import check_output
+
+import pytest
 
 import requests
+
 import retrying
 
+from pkgpanda.build import load_json
 from test_util.marathon import get_test_app, get_test_app_in_docker
 
 
+def config():
+    return load_json('/opt/mesosphere/etc/expanded.config.json')
+
+
+skip_if_lb_enabled = pytest.mark.skipif(config()['enable_lb'] == 'true', reason='Load Balancer enabled')
```

Comment:
My suggestion is to factor out 

```
def lb_enabled(): 
    return config()['enable_lb'] == 'true'
```

And similar for `lb_disabled`

and then you could use the `@pytest.mark.skipif(lb_enabled()` as a decorator for the tests instead of "factoring out the entire skipif object". 

The advantages I see are:

* This logic if is lb_enabled, lb_disabled can be used independent of using it with skipif
* The skipif decorator stays closer to the test as it is commonly used.